PIL/Image.py

```diff
@@ -1959,6 +1962,25 @@ def split(self):
                 ims.append(self._new(self.im.getband(i)))
         return tuple(ims)
 
+    def getchannel(self, channel):
```

Comment:
We use band elsewhere for just about everything related to channels, but channel is probably a more accepted name in the general graphics world.